ostia-operator/test/tests/common/common.sh

```diff
@@ -67,11 +73,23 @@ wait_for_pod_ready () {
 # Expects two args: Endpoint to make HTTP request against, number of requests to make
 do_http_get() {
  for i in $(seq 1 ${2}); do
-   curl -k -s -o /dev/null -w "%{http_code}\n" ${1};
+   curl -k -s -o /dev/null -w "%{http_code}\n" ${1}
  done | uniq -c
 
 }
 
+# Functionally equal to do_http_get but calling will execute within an apicast Pod in provided namespace
+# This function should be used when needed to simulate different source ip for testing
+# Relies on setting the host header
+# Expects three args: Host to make HTTP request against, number of requests to make, namespace
+do_http_get_in_pod() {
```

Comment:
Perhaps splitting this func into another one run_cmd_in_pod() ... could be useful